### DISCUSSION TITLE
Welcome to Nintendo Discussions!

### DISCUSSION THREAD

**Discussion by NicoHood** [NicoHood/Nintendo#32] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dur75945** [comment]
Probably a dumb question, but how far off is this project from a say Xbox One controller to GameCube adapter?

**Reply by NicoHood** [reply]
You can built that if you have another library that is able to read out the xbox controll (fast enough). There should be another thread where someone did that with PS3 controller, I think.

**Reply by dur75945** [reply]
like this? https://github.com/felis/USB_Host_Shield_2.0#xbox-one-library

**Reply by NicoHood** [reply]
Yes

**Reply by dur75945** [reply]
just taking a guess here, if I had an adapter using 2 separate Arduinos, it would be laggy right?

**Reply by dur75945** [reply]
wait, nvm. im stupid

**Reply by dur75945** [reply]
if my goal was to create an adapter that takes the inputs from a non gc controller (Xbox one or switch pro) and then recreate those inputs in a form that would be readable for any wii/gamecube with minimal latency, I would want all the code to be run on a single board. However when I was researching the Xbox library, I saw that a standalone board was being sold (http://shop.tkjelectronics.dk/product_info.php?products_id=43). In your opinion would it be more beneficial to have code running on a single board, or to make use of the board being sold above, which is meant to be used with a separate Arduino. (the latter being the easier of the 2 options)

**Reply by NicoHood** [reply]
The link provided is an usb host shield. It is not a standalone board, just a shield/addon.

**Reply by dur75945** [reply]
ok nvm, to reduce input lag I should obviously look for a 16mhz (the fastest on the market that I've seen) Arduino to use with the shield, but to your knowledge, is there a particular architecture i should be looking for, or is the more expensive Arduino going to be what I want, or does it not matter.

**Reply by NicoHood** [reply]
In that case I recommend using an arduino uno. That should work I guess
